feedback/utils.py

```diff
@@ -1,6 +1,44 @@
 """Utilities for feedback app"""
 
+# feedbackblock/feedback/utils.py
+from urllib.parse import urlencode, urlparse, urlunparse
+from opaque_keys.edx.keys import UsageKey
+from django.conf import settings
+
 
 def _(text):
     """Dummy `gettext` replacement to make string extraction tools scrape strings marked for translation"""
     return text
+
+
+def get_lms_link_for_item(location, preview=False):
+    """
+    Returns an LMS link to the course with a jump_to to the provided location.
+    """
+    assert isinstance(location, UsageKey)
+
+    try:
+        # pylint: disable=import-outside-toplevel
+        from openedx.core.djangoapps.site_configuration.models import SiteConfiguration
```

Comment:
generally speaking, it is a bad pattern for plugins to import code from edx-platform, as it creates coupling between something that is supposed to be stable (the plugin) and something that is inherently unstable (edx-platform modules). However, I don't know of any better way to let this block access SiteConfiguration other than adding an new XBlock runtime service, and I don't think a new SiteConfig runtime service is a good idea either given that we want to move away from SiteConfig. So, long story short, this is a bad pattern, but I'm OK with it for this PR given that there's no better way and we need a quick fix for this bug.